Fix the Arabic language in a string in strings.txt

Colonel-Dixie opened the pull request:
The Arabic language was wrong, as the script places the letters from left to right, while the Arabic language is written and read from right to left. And any Arabic reader will not understand it as it look like gibberish. I corrected the Arabic and made it right to left, and the name of the OpenStreetMaps was written in English instead of Arabic, which may have been the reason for reversing the entire Arabic text to start from the left.

biodranik reviewed (approved):
Thanks! Was the text looking wrong in Organic Maps app?

Colonel-Dixie commented:
> Was the text looking wrong in Organic Maps app?

I saw this error from a pull request a couple of days ago and it was the line above or under an edited one, that's why I saw it and fixed it. I did not see this error previously because my iPhone is set to English and the app was also in English.
I installed a fresh copy on an android VM on my computer and yes the error that I fixed in this pull requests is present in the latest version on f-droid. here is a screenshot, green is correct, red is incorrect 
![image](https://github.com/organicmaps/organicmaps/assets/35276833/ec9fde64-62e8-4efe-b5dc-6c9a7b51bfcc)
And when I continue I saw another error, (Have not fixed yet), see the blue button, also messed up:     
![image](https://github.com/organicmaps/organicmaps/assets/35276833/155a5bc0-a296-413d-b01d-730794436c64)
I will add another a fix pull request later fixing this and other errors maybe after a deep dive in the app this week in my free time.       
   
I also catched a very big issue with right-to-left thing couple of months ago and I just explained it in issue #6504     
   
   Thank you.
